src/NuGet.Clients/NuGet.PackageManagement.UI/PackageItemLoader.cs

```diff
@@ -192,6 +196,42 @@ public async Task UpdateStateAsync(IProgress<IItemLoaderState> progress, Cancell
             NuGetEventTrigger.Instance.TriggerEvent(NuGetEvent.PackageLoadEnd);
         }
 
+        private async Task<SearchResult<IPackageSearchMetadata>> CombineSearchAsync(
+            ContinuationToken continuationToken,
+            CancellationToken cancellationToken)
+        {
+            cancellationToken.ThrowIfCancellationRequested();
+            if (_packageFeed == null)
```

Comment:
You don't need this. 
There's a null check in the constructor of this class.

Reply:
Fixed